test/integration/save-settlement.test.js

```diff
@@ -0,0 +1,37 @@
+const db = require('../../app/data')
+
+jest.mock('../../app/inbound/get-completed-payment-request-by-invoice-number')
+const saveSettlement = require('../../app/inbound/return/save-settlement')
+
+let settlement
+
+describe('process save settlement', () => {
+  beforeEach(() => {
+    settlement = JSON.parse(JSON.stringify(require('../mock-settlement')))
+  })
+
+  afterEach(async () => {
+    jest.clearAllMocks()
+    await db.sequelize.truncate({
+      cascade: true,
+      restartIdentity: true
+    })
+  })
+
+  test('confirm settlement is saved in database when valid settlement is given', async () => {
+    await saveSettlement(settlement)
+    const result = await db.settlement.findOne()
+    expect(result).not.toBeNull()
+  })
+
+  test('confirm error is thrown and settlement is not saved in the database when invalid settlement is given', async () => {

```

Comment:
would split into 2 tests:

1. should throw (as you have minus lines 34 and 35)
2. should not save (instead of wrapper just have as plain try-catch)